workspaces/gamut-storybook/.storybook/decorators/wrapper.js

```diff
@@ -1,16 +1,9 @@
 import React from 'react';
-import { Container } from 'gamut';
-import '@codecademy/gamut-styles/core/index';
+import 'gamut-styles/core/index';
 import './styles';
 
-export const wrapper = story => (
-  <Container
-    justify="stretch"
-    column
-    style={{ minHeight: '100vh', padding: '1rem' }}
-  >
-    {story()}
-  </Container>
-);
+export const wrapper = story => {
+  return <>{story()}</>;
```

Comment:
Curiosity: why not just...

```suggestion
  return story()
```